server/app/repository/ReportingRepository.java

```diff
@@ -28,9 +28,19 @@ public ReportingRepository(Clock clock) {
     this.database = DB.getDefault();
   }
 
-  /** Loads data from the monthly reporting view. */
+  /**
+   * Loads data from the monthly reporting view. Does not include data from current month because
+   * the job to refresh the view is scheduled to run at the beginning of each month. For current
+   * month reporting data use {@code loadThisMonthReportingData}.
+   */
   public ImmutableList<ApplicationSubmissionsStat> loadMonthlyReportingView() {
-    return database.sqlQuery("SELECT * FROM monthly_submissions_reporting_view").findList().stream()
+    return database
+        .sqlQuery(
+            "SELECT * FROM monthly_submissions_reporting_view\n"
```

Comment:
Do we have a preference between using the ORM methods vs manual queries, when the ORM would support the query?

Reply:
We do have a preference for using the ORM. In this case I had a hard time getting the ORM to behave with this query for some reason so I used raw SQL.